api_tests/src/user.spec.ts

```diff
@@ -19,8 +19,13 @@ import {
   API,
   resolveComment,
   saveUserSettingsFederated,
+  setupLogins,
 } from './shared';
 
+beforeAll(async () => {
+  await setupLogins();
+});
```

Comment:
For some reason beta.auth ends up being None in this file, no clue why. That causes the user settings test to fail, because `resolvePerson(beta, apShortname)` counts as unauthenticated, and doesnt fetch the actor over network.